docs/src/methods/fourier_surrogates.md

```diff
@@ -43,10 +48,38 @@ s = surrogate(ts, PartialRandomization(0.5))
 surroplot(ts, s)
 ```
 
+In addition to [`PartialRandomization`](@ref), we provide two other algorithms for producing partially randomized surrogates, outlined below.
+
+### Relative partial randomization
+
+The [`PartialRandomization`](@ref) algorithm corresponds to assigning entirely new phases to the Fourier spectrum with some degree of randomness, regardless of any deterministic structure in the original phases. As such, even for $\alpha = 0$ the surrogate time series can differ drastically from the original time series.
+
+By contrast, the [`RelativePartialRandomization`](@ref) procedure draws phases as:
+
+$$\phi \to \phi + \alpha \xi, \quad \xi \sim \mathcal{U}(0, 2\pi).$$
+
+With this algorithm, phases are progressively corrupted by higher values of $\alpha$: surrogates are identical to the original time series for $\alpha = 0$, equivalent to random noise for $\alpha = 1$, and retain some of the structure of the original time series when $0 < \alpha < 1$. This procedure is particularly useful for controlling the degree of chaoticity and non-linearity in surrogates of chaotic systems.
+
+### Spectral partial randomization
+
+Both of the algorithms above randomize phases at all frequency components to the same degree.
+To assess the contribution of different frequency components to the structure of a time series, the [`SpectralPartialRandomization`](@ref) algorithm only randomizes phases above a frequency threshold.
+The threshold is chosen as the lowest frequency at which the power spectrum of the original time series drops below a fraction $1-\alpha$ of its maximum value (such that the power contained above the frequency threshold is a proportion $\alpha$ of the total power, excluding the zero frequency).
+
+See the figure below for a comparison of the three partial randomization algorithms:
+```@example MAIN
+x = open(joinpath(@__DIR__, "../../../src/plotting/lorenz_timeseries.csv"), "r") do f # hide
+    f |> readlines .|> Meta.parse # hide
```

Comment:
Please don't do that. Adding CSV files makes the code heavy. And there is no reason. Use the package DynamicalSystemsBase.jl (which we anyways use in the docs somewhere else) and construct a timeseries by defining and evolving the Lorenz system.